valuesets/ValueSet-England-RoleCode.xml

```diff
@@ -0,0 +1,28 @@
+<ValueSet xmlns="http://hl7.org/fhir">
```

Comment:
Add <?xml version="1.0" encoding="utf-8"?> at the top of the code

Reply:
Changes have been made